### PR TITLE
net-misc/gupnp-tools: add upstream commit as a patch to fix bug #598284

### PR DESCRIPTION
**Pull request by kobboi**

### CODE REVIEW

File: net-misc/gupnp-tools/gupnp-tools-0.8.11.ebuild [hunk 1]
```diff
@@ -30,3 +30,11 @@ DEPEND="${RDEPEND}
 	sys-devel/gettext
 	virtual/pkgconfig
 "
+
+src_prepare() {
+	# Fixes 598284 with upstream patch
+	# https://git.gnome.org/browse/gupnp-tools/commit/?id=e9ec9634207e4c2eea6d268ee29b57e687c1f178
+	epatch "${FILESDIR}"/fix-compiling-when-using-recent-gupnp-av.patch
+	gnome2_src_prepare
```
Comment (SoapGentoo) [inline net-misc/gupnp-tools/gupnp-tools-0.8.11.ebuild]: consider whether `EAPI=6` bumping + `PATCHES` isn't easier than beating the dead `utils` horse

**Comment by kobboi** [comment]
Modified 0.8.12 to use PATCHES. Dropping 0.8.11 in a separate commit.

**Comment by kobboi** [comment]
Fixed QA issue of not regenerating the Manifest after deleting the ebuild

**Comment by kobboi** [comment]
In the mean time, 0.8.13 is out which includes this patch :|

**Pull request closed without merging**